src/utils/query-string.js

```diff
@@ -1,7 +1,10 @@
 export function buildQueryString(params) {
-    const entries = Object.keys(params).map(
-        key => encodeURIComponent(key) + '=' + encodeURIComponent(params[key])
-    );
+    const entries = Object.keys(params)
+        .sort()
```

Comment:
Sorting here prevents the url params from flashing by adding some more consistency